embedly/embedly.py

```diff
@@ -54,13 +52,21 @@ def get_urls_from_embedly(urls):
     except requests.RequestException:
         response = None
 
+    embedly_data = []
     if response is not None:
         try:
-            response_data = json.loads(response.content)
+            embedly_data = json.loads(response.content)
         except (TypeError, ValueError):
             pass
```

Comment:
@ncloudioj yep exactly, we have no error logging capacity right now but when we do #13 these exceptions will be logged and also reported to the client through #22